quay-enterprise/geo-replication.md

```diff
@@ -0,0 +1,36 @@
+# Georeplication of storage in Quay Enterprise
```

Comment:
mixed case. below is title case, this is sentence. I think the official style guide is sentence case